bundles/org.openhab.binding.folderwatcher/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -123,4 +123,50 @@
 			</parameter>
 		</config-description>
 	</thing-type>
+	<thing-type id="s3bucket">
+		<label>AWS S3 Bucket</label>
+		<description>AWS S3 bucket to be watched</description>
+
+		<channels>
+			<channel id="newfile" typeId="newfile-channel"/>
+		</channels>
+
+		<config-description>
+			<parameter name="s3BucketName" type="text" required="true">
+				<label>S3 Bucket Name</label>
+				<description>Name of the S3 bucket to be watched</description>
+			</parameter>
+			<parameter name="s3Path" type="text">
+				<label>S3 Path</label>
+				<description>S3 path (folder) to be monitored</description>
+			</parameter>
+			<parameter name="awsRegion" type="text" required="true">
+				<label>AWS Region</label>
+				<description>AWS Region of S3 bucket</description>
```

Comment:
Missed this previously, now spotted when checking the corresponding chapter in the README:
```suggestion
				<description>AWS region of S3 bucket</description>
```